bld/listDefaultNamelist.pl

```diff
@@ -270,11 +270,17 @@ sub GetListofNeededFiles {
 
            my @bgcsettings   = $definition->get_valid_values( "bgc_mode", 'noquotes'=>1 );
```

Comment:
I'm not sure if this is a useful script anymore. We should talk about getting rid of it.

Reply:
Ok, let me know.

Reply:
Meanwhile I also realized that my mods in this script were unnecessary, so I have now reverted them.

Reply:
Referenced this conversation in a new issue to ensure it didn't get lost.